### PR TITLE
refactor: use ColumnAdjuster from utils repo

### PR DESCRIPTION
**Pull request by haxxmaxx**

### CODE REVIEW

**Comment by haxxmaxx** [comment]
slight drop in coverage, but there is no logic to test really

File: src/ext/property-panel/data.js [hunk 1]
```diff
@@ -208,12 +208,12 @@ const getColumnResize = {
...
     expression: "optional",
     defaultValue: 20,
-    show: (data) => data.qDef.columnWidth?.type === ColumnWidthTypes.PERCENTAGE,
+    show: (data) => data.qDef.columnWidth?.type === ColumnWidthType.Percentage,
     change(data) {
       data.qDef.columnWidth.percentage =
         data.qDef.columnWidth.percentage === undefined
           ? data.qDef.columnWidth.percentage
-          : Math.max(1, Math.min(MAX_COLUMN_PERCENTAGE_WIDTH, data.qDef.columnWidth.percentage));
+          : Math.max(1, Math.min(ColumnWidthValues.PixelsMax, data.qDef.columnWidth.percentage));
```
Comment (cbt1) [inline src/ext/property-panel/data.js:216]: `ColumnWidthValues.PixelsMax`?

File: package.json [hunk 2]
```diff
@@ -114,7 +114,7 @@
     "@playwright/test": "1.39.0",
     "@qlik-trial/sprout": "3.7.1",
     "@qlik/eslint-config": "0.4.30",
-    "@qlik/nebula-table-utils": "^2.6.1",
+    "@qlik/nebula-table-utils": "2.6.1",
```
Comment (a-m-dev) [inline package.json:117]: I noticed this one as well, and addressed in my PR too

**Review by a-m-dev — APPROVED** [review]
🚀

File: package.json (hunk 2 above)
Comment (a-m-dev) [reply package.json:117]: this one is in now

File: src/ext/property-panel/data.js (hunk 1 above)
Comment (haxxmaxx) [reply src/ext/property-panel/data.js:216]: that is indeed wrong, I think I need to test this a bit more in the client setting values using the pp.

**Pull request merged**